crates/runtime/src/http/v1/mod.rs

```diff
@@ -231,6 +242,10 @@ pub async fn to_http_response(
 
     let mut headers = HeaderMap::new();
 
+    if let Some(header_value) = format.to_accept_header() {
+        headers.insert(CONTENT_TYPE, header_value);
+    };
```

Comment:
```suggestion
    }
```

oops, wrong one earlier!